modules/integration/src/test/scala/scala/cli/integration/TestUtil.scala

```diff
@@ -102,10 +102,21 @@ object TestUtil {
             helper(count + 1)
           }
       }
-    helper(if (isCI) 1 else maxAttempts)
+    helper(1)
   }
 
+  def retryOnCi[T](maxAttempts: Int = 3, waitDuration: FiniteDuration = 5.seconds)(
+    run: => T
+  ) = retry((if (isCI) 1 else maxAttempts), waitDuration)(run)
```

Comment:
I think it's the opposite here
```suggestion
  ) = retry((if (isCI) maxAttempts else 1), waitDuration)(run)
```